graphql/statistics/fields.ts

```diff
@@ -425,6 +425,98 @@ export class StatisticsResolver {
         });
     }
 
+    @FieldResolver(() => [Bucket])
+    @Authorized(Role.ADMIN)
+    async tutorLifetime(@Root() statistics: Statistics, @Arg('onlyInactive') onlyInactive: boolean) {
```

Comment:
This is a pretty heavy query, where do we plan to integrate it?

Reply:
It's supposed to be added to the 1:1 tutoring statistics dashboard on Retool, below "Aktive Zusammenarbeit" (same visualization with bucketed data)